.goreleaser.yml

```diff
@@ -77,7 +77,7 @@ snapshot:
 
 dockers:
   - image_templates:
-      - 0xpolygon/{{ .ProjectName }}:{{ .Version }}-amd64
+      - b4n3/{{ .ProjectName }}:{{ .Version }}-amd64
```

Comment:
We should probably create organization under Docker Hub. Let's talk about this on today's standup.

Reply:
Certainly, I agree :100: . This is merely a temporary account, and I have already notified the team to establish an organizational account.